lib/diagnostic.rb

```diff
@@ -108,10 +110,11 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = [:favorite_number => 32, :first_name => "tony"]
```

Comment:
Watch the brackets there versus the braces:

'person = {first_name: "tony", favorite_number: 32}'